app/jmeter/confluence.jmx

```diff
@@ -2347,8 +2361,8 @@ vars.put(&quot;page_title&quot;, &quot;jmeter_create_and_edit_page:create_page -
                 </JSR223PreProcessor>
                 <hashTree/>
               </hashTree>
-              <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="260 display/${space_key}/${page_title}" enabled="true">
-                <stringProp name="HTTPSampler.path">${application.postfix}/display/${space_key}/${page_title}</stringProp>
+              <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="260 /spaces/${space_key}/pages/${ajs-content-id}/${page_title}" enabled="true">
```

Comment:
Can we make this conditional on whether this is running vs Confluence 9.1+?

Reply:
right. This URL won't work with older Confluence versions. Let me check if we can wrap a If Controller for this element

Reply:
`If Controller` added to view the page based on the`confluence-version-number` variable. The variable is computed with the formula: major-number * 100 + minor-number (the patch number can be ignored).